src/index.js

```diff
@@ -1,13 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] `optional`
- [ ] Please enable the disabled linters because it defeats the purpose of installing linters

Reply:
I missed that.. I'll fix it.
Thanks!